pennylane/ops/cv.py

```diff
@@ -973,15 +973,15 @@ class PolyXP(CVObservable):
 
     **Details:**
 
-    * Number of wires: None (applied to the entire system)
+    * Number of wires: Any
     * Number of parameters: 1
     * Observable order: 2nd order in the quadrature operators
     * Heisenberg representation: :math:`A`
 
     Args:
         q (array[float]): expansion coefficients
     """
-    num_wires = 0
+    num_wires = Any
```

Comment:
Do we have built-in tests to confirm this? Right now, PolyXP acts on 1 wire and the whole system has to have one wire. If you call PolyXP on one wire from a system of two wires, the error is not very descriptive of where the problem is coming from.

Reply:
Good catch, let me double check this!

Reply:
Good eye @AroosaIjaz. I've pushed a fix that now allows PolyXP to act on any number of wires (this already worked in the SF plugin, but not in the `default.gaussian` plugin). Can you verify that it works for you now?

Reply:
Hey @josh146 , I am sorry I did not get to this yet
will try to get it done today!

Reply:
@josh146, its working nicely now :) PolyXP is acting on any wire we specify!

Reply:
Perfect, thanks @AroosaIjaz!